tools/perf/multi-node/wrk.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: wrk
+  name: wrk
+  namespace: kuma-demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: wrk
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: wrk
+    nodeName: kind-worker2
+    spec:
+      containers:
+      - image: lukdzied/wrk:latest
```

Comment:
it this `wrk2`? also should we publish this in `kuma`?

Reply:
It's `wrk`, I haven't used `wrk2` because I didn't want to test constant throughput but the highest possible on this VM, and `wrk2` requires number of requests.

Reply:
I'm ok with an official WRK image but I think there aren't any "official" sources. Just for the future we shouldn't start failing if you remove the image from dockerhub.